### PR TITLE
tutorial: update of server URL and various outputs

### PR DESCRIPTION
**Pull request by neteler**
**! This is currently a WIP, please do not merge just now. !**

The tutorial at https://actinia.mundialis.de/tutorial/tutorial_data_access.html is a bit outdated.

This PR contains needed fixes:
- update of ACTINIA_URL
- update from non-existing "superadmin" to "demouser"
- update of JSON outputs

### CODE REVIEW

**Comment by neteler** [comment]
To be discussed: the tutorial uses "superadmin", hence is only usable on local installations and not https://actinia.mundialis.de/
--> add hints on "demouser"?

File: docs/index.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ The full API documentation is available here: https://actinia.mundialis.de/api_d
    Actinia database access <tutorial_data_access>
    Time-series sampling <tutorial_strds_sampling>
    Landsat NDVI computation <tutorial_landsat_ndvi>
-   Sentinel2A NDVI computation <tutorial_sentinel2a_ndvi>
+   Sentinel-2A NDVI computation <tutorial_Sentinel-2A_ndvi>
```
Comment (AnikaBettge) [inline docs/index.rst]: Please verify

File: docs/installation.rst [hunk 2]
```diff
@@ -35,13 +35,11 @@ First install Proj4 and GRASS GIS by downloading the latest svn version and comp
 
         cd /tmp
 
-        wget https://grass.osgeo.org/grass75/source/snapshot/grass-7.5.svn_src_snapshot_2018_03_10.tar.gz
-        tar xzvf grass-7.5.svn_src_snapshot_2018_03_10.tar.gz
-        mv grass-7.5.svn_src_snapshot_2018_03_10 grass_trunk
+        wget https://grass.osgeo.org/grass76/source/snapshot/grass-7.6.git_src_snapshot_2019_07_27.tar.gz
```
Comment (AnikaBettge) [inline docs/installation.rst]: Please update to grass78

File: docs/installation.rst [hunk 3]
```diff
@@ -140,7 +138,7 @@ First install Proj4 and GRASS GIS by downloading the latest svn version and comp
 
     ..
 
-5. GRASS GIS requires a Python2 environment for running, so we create one.
+5. GRASS GIS 7.6 requires a Python2 environment for running, so we create one.
```
Comment (AnikaBettge) [inline docs/installation.rst]: idem

File: docs/tutorial_landsat_ndvi.rst [hunk 4]
```diff
@@ -322,7 +322,7 @@ the created NDVI GeoTiff file as well as univariate statistics of the computed N
             {
               "executable": "python2",
               "parameter": [
-                "/usr/local/bin/grass75",
+                "/usr/local/bin/grass76",
```
Comment (AnikaBettge) [inline docs/tutorial_landsat_ndvi.rst]: grass78
python2?

File: docs/tutorial_process_chain.rst [hunk 5]
```diff
@@ -107,13 +107,13 @@ Then slope and aspect are computed with *r.slope.aspect* and specified for expor
 
 .. rubric:: Footnotes
 
-.. [#grassmodulelist] https://grass.osgeo.org/grass74/manuals/index.html
+.. [#grassmodulelist] https://grass.osgeo.org/grass76/manuals/index.html
```
Comment (AnikaBettge) [inline docs/tutorial_process_chain.rst]: grass78

File: docs/tutorial_sentinel2a_ndvi.rst [hunk 6]
```diff
@@ -177,7 +177,7 @@ the created NDVI GeoTiff file as well as univariate statistics of the computed N
             {
               "executable": "python2",
               "parameter": [
-                "/usr/local/bin/grass75",
+                "/usr/local/bin/grass76",
```
Comment (AnikaBettge) [inline docs/tutorial_sentinel2a_ndvi.rst]: grass78

File: docs/index.rst (hunk 1 above)
Comment (neteler) [inline docs/index.rst]:
```suggestion
   Sentinel-2A NDVI computation <tutorial_sentinel2a_ndvi>
```

File: docs/installation.rst (hunk 2 above)
Comment (neteler) [inline docs/installation.rst]:
```suggestion
        wget https://grass.osgeo.org/grass78/source/snapshot/grass-7.8.git_src_snapshot_latest.tar.gz
```

File: docs/installation.rst [hunk 7]
```diff
@@ -35,13 +35,11 @@ First install Proj4 and GRASS GIS by downloading the latest svn version and comp
 
         cd /tmp
 
-        wget https://grass.osgeo.org/grass75/source/snapshot/grass-7.5.svn_src_snapshot_2018_03_10.tar.gz
-        tar xzvf grass-7.5.svn_src_snapshot_2018_03_10.tar.gz
-        mv grass-7.5.svn_src_snapshot_2018_03_10 grass_trunk
+        wget https://grass.osgeo.org/grass76/source/snapshot/grass-7.6.git_src_snapshot_2019_07_27.tar.gz
+        tar xzvf grass-7.6.git_src_snapshot_2019_07_27.tar.gz
```
Comment (neteler) [inline docs/installation.rst]:
```suggestion
        tar xzvf grass-7.8.git_src_snapshot_latest.tar.gz
```

File: docs/installation.rst [hunk 8]
```diff
@@ -35,13 +35,11 @@ First install Proj4 and GRASS GIS by downloading the latest svn version and comp
...
+        mv grass-7.6.git_src_snapshot_2019_07_27 grass_master
```
Comment (neteler) [inline docs/installation.rst]:
```suggestion
        mv grass-7.8.git_src_snapshot_????_??_?? grass_master
```

File: docs/installation.rst [hunk 9]
```diff
@@ -35,13 +35,11 @@ First install Proj4 and GRASS GIS by downloading the latest svn version and comp
...
 
-        cd grass_trunk
-        svn cleanup
-        svn up
+        cd grass_master
 
         export INTEL="-march=native -std=gnu99 -fexceptions -fstack-protector -m64"
```
Comment (neteler) [inline docs/installation.rst]:
```suggestion
        export INTEL="-std=gnu99 -fexceptions -fstack-protector -m64"
```

File: docs/installation.rst (hunk 3 above)
Comment (neteler) [inline docs/installation.rst]:
```suggestion
5. GRASS GIS 7.8 requires a Python 3 environment for running, so we create one (not needed if only Python 3 is present on the machine).
```

File: docs/tutorial_landsat_ndvi.rst [hunk 10]
```diff
@@ -322,7 +322,7 @@ the created NDVI GeoTiff file as well as univariate statistics of the computed N
             {
               "executable": "python2",
```
Comment (neteler) [inline docs/tutorial_landsat_ndvi.rst]:
```suggestion
              "executable": "python3",
```

File: docs/tutorial_landsat_ndvi.rst (hunk 4 above)
Comment (neteler) [inline docs/tutorial_landsat_ndvi.rst]:
```suggestion
                "/usr/local/bin/grass78",
```

File: docs/tutorial_process_chain.rst (hunk 5 above)
Comment (neteler) [inline docs/tutorial_process_chain.rst]:
```suggestion
.. [#grassmodulelist] https://grass.osgeo.org/grass78/manuals/index.html
```

File: docs/tutorial_process_chain.rst [hunk 11]
```diff
@@ -107,13 +107,13 @@ Then slope and aspect are computed with *r.slope.aspect* and specified for expor
...
 .. [#grassmodule] https://actinia.mundialis.de/api_docs/#/definitions/GrassModule
 .. [#inputs] https://actinia.mundialis.de/api_docs/#/definitions/InputParameter
 .. [#outputs] https://actinia.mundialis.de/api_docs/#/definitions/OutputParameter
-.. [#rlopeaspect] https://grass.osgeo.org/grass74/manuals/r.slope.aspect.html
-.. [#mapset] https://grass.osgeo.org/grass74/manuals/grass_database.html
-.. [#gregion] https://grass.osgeo.org/grass74/manuals/g.region.html
+.. [#rlopeaspect] https://grass.osgeo.org/grass76/manuals/r.slope.aspect.html
```
Comment (neteler) [inline docs/tutorial_process_chain.rst]:
```suggestion
.. [#rlopeaspect] https://grass.osgeo.org/grass78/manuals/r.slope.aspect.html
```

File: docs/tutorial_process_chain.rst [hunk 12]
```diff
@@ -107,13 +107,13 @@ Then slope and aspect are computed with *r.slope.aspect* and specified for expor
...
+.. [#mapset] https://grass.osgeo.org/grass76/manuals/grass_database.html
```
Comment (neteler) [inline docs/tutorial_process_chain.rst]:
```suggestion
.. [#mapset] https://grass.osgeo.org/grass78/manuals/grass_database.html
```

File: docs/tutorial_process_chain.rst [hunk 13]
```diff
@@ -107,13 +107,13 @@ Then slope and aspect are computed with *r.slope.aspect* and specified for expor
...
+.. [#gregion] https://grass.osgeo.org/grass76/manuals/g.region.html
```
Comment (neteler) [inline docs/tutorial_process_chain.rst]:
```suggestion
.. [#gregion] https://grass.osgeo.org/grass78/manuals/g.region.html
```

File: docs/tutorial_sentinel2a_ndvi.rst [hunk 14]
```diff
@@ -177,7 +177,7 @@ the created NDVI GeoTiff file as well as univariate statistics of the computed N
             {
               "executable": "python2",
```
Comment (neteler) [inline docs/tutorial_sentinel2a_ndvi.rst]:
```suggestion
              "executable": "python3",
```

File: docs/tutorial_sentinel2a_ndvi.rst (hunk 6 above)
Comment (neteler) [inline docs/tutorial_sentinel2a_ndvi.rst]:
```suggestion
                "/usr/local/bin/grass78",
```

**Comment by neteler** [comment]
Merging now.
For sure more work is needed but we need to have the updated version online now.

**Comment by neteler** [comment]
Note: only visible after next redeployment of actinia.mundialis.de (without cache).

**Pull request merged**